scripts/processing/05-run_ica.py

```diff
@@ -31,7 +31,7 @@ def run_ica(subject_id, tsss=None):
                                 % (run, tsss))
         else:
             run_fname = op.join(data_path, 'run_%02d_filt_sss_highpass-%sHz'
-                                '_raw.fif' % (run, 1))
```

Comment:
umm ... this is fine I think. You always want to use a highpassed version of your data for ICA because otherwise ICA crashes